src/Middleware/Diagnostics/src/ExceptionHandler/ExceptionHandlerServiceCollectionExtensions.cs

```diff
@@ -23,6 +23,7 @@ public static IServiceCollection AddExceptionHandler(this IServiceCollection ser
         ArgumentNullException.ThrowIfNull(services);
         ArgumentNullException.ThrowIfNull(configureOptions);
 
+        services.AddProblemDetailsExceptionHandler();
```

Comment:
Assuming that [`AddProblemDetails` ](https://github.com/dotnet/aspnetcore/blob/70048beee894074e9718206a0017d9b080ee66ff/src/Http/Http.Extensions/src/ProblemDetailsServiceCollectionExtensions.cs#L20) is called, and since the `ProblemDetailsExceptionHandler` always handle the exception, by adding the `ProblemDetailsExceptionHandler` in this method, there should be doc indicating that if user wants to have its own `IExceptionHandler` they have to call `AddExceptionHander<TExceptionHandler>` before `AddExceptionHandler(this IServiceCollection services, Action<ExceptionHandlerOptions> configureOptions)` in order to give user's `IExceptionHandler`  a chance to handle the exception.